docs/application/tizen-studio/release-notes/2-4-release-notes.md

```diff
@@ -0,0 +1,86 @@
+# Tizen Studio 2.4 Release Notes
+
+
+## IDE and Tools
+
+
+### New Features
+
+-   Device Manager
+	-   Enable Device Manager Context UI to be extended by plugins
+-   RT-IDE
+	-   RT-IDE GCC Toolchain update to 6.3.1
+-   SmartThings
+	-   Make setupId mandatory field and map it to device onboarding id
+	-   Updated ramls for smart things resources
+-   Package Manager
+	-   Remove *.po files getting packaged into tpk
+
+### Changed Features
+-	RT-IDE
+	-   Removed local template for creating project. Project can be created using latest RT code from github.
```

Comment:
Removed the local template for creating a project. The project can be created using the latest RT code from GitHub.